wsmaster/che-core-api-devfile/src/main/java/org/eclipse/che/api/devfile/server/convert/DevfileConverter.java

```diff
@@ -139,11 +141,10 @@ public WorkspaceConfigImpl devFileToWorkspaceConfig(
       applier.apply(config, component, contentProvider);
     }
 
-    devfile
-        .getProjects()
-        .stream()
-        .map(projectConverter::toWorkspaceProject)
-        .forEach(project -> config.getProjects().add(project));
+    for (Project project : devfile.getProjects()) {
```

Comment:
Just wondering if there any technical reason to do this change?

Reply:
Yes, because `toWorkspaceProject` now throws a checked exception and therefore cannot be used as a lambda in `map`.